src/components/codeContext.tsx

```diff
@@ -123,38 +128,68 @@ const formatApiUrl = ({scheme, host}: Dsn) => {
   return `${scheme}${apiHost}/api`;
 };
 
+function getHost(): string {
+  if (process.env.NODE_ENV === 'development') {
+    return 'http://dev.getsentry.net:8000';
```

Comment:
I'm not familiar with dev.getsentry.net - what's the reason for pointing to that in develop?

Reply:
`dev.getsentry.net` is the canonical hostname for the getsentry devserver. The docs support endpoint is only available in getsentry.